README.md

```diff
@@ -678,50 +1352,77 @@ To get the current page count of the document.
 
 Returns a Promise.
 
+Promise Parameters:
+
+Name | Type | Description
+--- | --- | ---
+pageCount | int | the current page count of the document
+
 ```js
 this._viewer.getPageCount().then((pageCount) => {
   console.log('pageCount', pageCount);
 });
 ```
 
 ##### importAnnotations
-To import XFDF string to the current document.
+To import XFDF annotation string to the current document.
+
+Parameters:
+
+Name | Type | Description
+--- | --- | ---
+xfdf | string | annotation string in XFDF format for import
 
 Returns a Promise.
 
 ```js
-const xfdf = '<?xml version="1.0" encoding="UTF-8"?>\n<xfdf xmlns="http://ns.adobe.com/xfdf/" xml:space="preserve">...</xfdf>';
+const xfdf = '<?xml version="1.0" encoding="UTF-8"?>\n<xfdf xmlns="http://ns.adobe.com/xfdf/" xml:space="preserve">\n\t<annots>\n\t\t<circle style="solid" width="5" color="#E44234" opacity="1" creationdate="D:20190729202215Z" flags="print" date="D:20190729202215Z" page="0" rect="138.824,653.226,236.28,725.159" title="" /></annots>\n\t<pages>\n\t\t<defmtx matrix="1.333333,0.000000,0.000000,-1.333333,0.000000,1056.000000" />\n\t</pages>\n\t<pdf-info version="2" xmlns="http://www.pdftron.com/pdfinfo" />\n</xfdf>';
 this._viewer.importAnnotations(xfdf);
 ```
 
 ##### exportAnnotations
 To extract XFDF from the current document.
 
-Perameters:
+Parameters:
 
 Name | Type | Description
 --- | --- | ---
-options | object | key: annotList, type: array
+options | object | key: annotList, type: array. If specified, annotations with the matching id and pageNumber will be exported; otherwise, export all annotations in the current document
```

Comment:
```suggestion
options | object | key: annotList, type: array. If specified, annotations with the matching id and pageNumber will be exported; otherwise, all annotations in the current document will be exported.
```